src/standalone/Soil/Runoff/Runoff.jl

```diff
@@ -136,21 +136,26 @@ function update_runoff!(
     t,
     model::AbstractSoilModel,
 )
-    column_integral_definite!(p.soil.h∇, is_saturated(Y, model))
+
+    ϑ_l = Y.soil.ϑ_l
+    FT = eltype(ϑ_l)
+    θ_i = :θ_i ∈ propertynames(Y.soil) ? Y.soil.θ_i : FT(0) # is this OK? can also make two methods.
```

Comment:
I would suggest turning this into a function whose name clearly says what is happening here. Also, `zero(FT)` is (marginally) better than `FT(0)`.